assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,54 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates a module on the server.
+	 *
+	 * Activate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
```

Comment:
@felixarntz we should probably change the signature of the functions here to match the underlying action creator rather than spreading the args through which is forcing us to document them differently.
What do you think?

Reply:
I agree, let's match the function signatures. Spreading them through is rather detrimental here.

Reply:
Great, matching the function signatures makes is much clearer and more simple for me.